<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerializerEditDlg</class>
 <widget class="QDialog" name="SerializerEditDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>216</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serializer Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QCheckBox" name="enableSerializationCHB">
     <property name="text">
      <string>Allow Serialization</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="outputGB">
     <property name="title">
      <string>Output Properties</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Output Frequency</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="frequencySB">
          <property name="maximum">
           <number>100000000</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>File Format</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2">
         <widget class="QComboBox" name="fileFormatCB">
          <item>
           <property name="text">
            <string>text</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>binary</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="multipleDirCHB">
        <property name="text">
         <string>Allow multiple restart snapshots</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="enableRestartCHB">
     <property name="text">
      <string>Enable Restart</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPB">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPB">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okPB</sender>
   <signal>clicked()</signal>
   <receiver>SerializerEditDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelPB</sender>
   <signal>clicked()</signal>
   <receiver>SerializerEditDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableSerializationCHB</sender>
   <signal>toggled(bool)</signal>
   <receiver>outputGB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>19</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
